src/commands/initProjectForVSCode/verifyVSCodeConfig.ts

```diff
@@ -77,20 +78,22 @@ export async function verifyInitForVSCode(actionContext: IActionContext, fsPath:
     return [<ProjectLanguage>language, <ProjectRuntime>runtime];
 }
 
-async function promptToInitializeProject(workspacePath: string): Promise<boolean> {
+async function promptToInitializeProject(workspacePath: string, actionContext: IActionContext): Promise<void> {
     const settingKey: string = 'showProjectWarning';
     if (getFuncExtensionSetting<boolean>(settingKey)) {
+        actionContext.properties.verifyConfigPrompt = 'initProject';
```

Comment:
Is this property effectively replacing `actionContext.properties.isInitialized = 'true';`?